bayesian/api_v1.py

```diff
@@ -1027,7 +1027,7 @@ def post():
 
         input_json = request.get_json()
         persist = request.args.get('persist', 'false') == 'true'
-
+        check_license = request.args.get('check_license', 'false') == 'true'
```

Comment:
@sara-02 I guess you want these StackAnalyses API, not in dependency-editor APIs.